db/seeds/overviews.rb

```diff
@@ -1,19 +1,16 @@
 # Copyright (C) 2012-2023 Zammad Foundation, https://zammad-foundation.org/
 
 overview_role = Role.find_by(name: 'Agent')
+
 Overview.create_if_not_exists(
-  name:      __('My Assigned Tickets'),
-  link:      'my_assigned',
+  name:      __('Unassigned tickets'),
+  link:      'all_unassigned',
```

Comment:
@YetAnotherGerrit Until now, we used [title case](https://en.wikipedia.org/wiki/Title_case) for the strings, as they usually appear on headings. Did you change that on purpose?

Regarding the old strings, you can probably just add a list of legacy strings at the end of the file with a leading comment line before them. The strings will do nothing there, but the extractor will find them.

Reply:
> Did you change that on purpose?

No, actually, it just felt more right to me. But I received multiple feedbacks already that they like "title case" more. I will update my PR back to title case. For me, the conceptional change is the important thing.

I will add the legacy string in the next commit as well. I will also update the translation catalog, so the CI-link runs successfully.

Reply:
Can we bring this forward in some way?

Reply:
Currently this is holding me back:
https://community.zammad.org/t/reduce-file-name-length-of-vcr-cassettes-files/12336

There are workarounds, but they are time consuming to discover.